internal/util/util.go

```diff
@@ -151,6 +161,90 @@ func KernelVersion() (string, error) {
 	return strings.TrimRight(string(utsname.Release[:]), "\x00"), nil
 }
 
+// KernelVersion holds kernel related informations
```

Comment:
This could be it's own PR that can get reviewed+merged quicker

Reply:
@nixpanic as no other PR is dependent on this one, as it's in a separate commit lets get it done here only.

Reply:
sure, thats fine